### PR TITLE
docs(README): mention immediate: false explicitly, improve formatting

### PR DESCRIPTION
**Pull request by JuroUhlar**

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  ...commonDxTeamPrettierConfig,
+  overrides: [
+    {
+      files: '*.md',
+      options: {
+        printWidth: 80,
+        singleQuote: true,
+      },
+    },
+  ],
```
Comment (TheUnderScorer) [inline .prettierrc.js:14]: Question - would it make sense to include this in the shared prettier configuration?

**Review by TheUnderScorer — DISMISSED** [review]

File: .prettierrc.js (hunk 1 above)
Comment (JuroUhlar) [reply .prettierrc.js:14]: I think so! I can make a PR there as well.

Comment (JuroUhlar) [reply .prettierrc.js:14]: No idea if I did this right, review appreciated: https://github.com/fingerprintjs/dx-team-toolkit/pull/60

Comment (TheUnderScorer) [reply .prettierrc.js:14]: Yep, looks good. Thanks!

Once the PR is merged, let's revert this change here 🙂

Comment (JuroUhlar) [reply .prettierrc.js]: @TheUnderScorer I upgraded the dependency and reverted the change, thank you!

Comment (JuroUhlar) [reply .prettierrc.js]: Any idea why the `lint` job is suddenly failing? It works fine for me locally. I tried to fix it with a flag but failed

Comment (TheUnderScorer) [reply .prettierrc.js]: It could be a issue with a lockfile. If it passes now, then probably all is good. Thanks @JuroUhlar!

**Review by TheUnderScorer — APPROVED** [review]

**Pull request merged**